### DISCUSSION TITLE
DNS泄露严重，是我设置的问题吗

### DISCUSSION THREAD

**Discussion by AEDELSTAN** [vvbbnn00/WARP-Clash-API#165] [Q&A] [answered]
如题，用IPCheck.ing测试，4 个DNS检测出口都泄露了（部分时候 ③ ④ 出口不泄露），分流测试ip全为 CN
同一网络环境下使用另一个机场订阅（自带规则）可以完美隐藏，包括分流测试ip都不是 CN
我已在规则内加入IPCheck.ing，生成的  选优+随机名+完整订阅 

![屏幕截图 2024-03-29 194212](https://github.com/vvbbnn00/WARP-Clash-API/assets/73707517/3cbd7fd0-b697-4671-b5b9-33d1a20b7f68)

本人小白，不清楚以上操作是否正确，望大佬们解答！

**Comment by AEDELSTAN** [comment, marked as answer]
已解决

是完整订阅的规则导致的
没有仔细研究自带规则，但是参照网上一些规则配置修改后，就没有DNS泄露了

**Reply by a414166402** [reply]
我也遇到了 请问您这边能否提供没内存泄露的配置模板？
